packages/prisma/zod-utils.ts

```diff
@@ -58,6 +58,20 @@ export const recurringEventType = z
   })
   .nullable();
 
+// dayjs iso parsing is very buggy - cant use :( - turns ISO string into Date object
+export const iso8601 = z.string().transform((val, ctx) => {
```

Comment:
We should have some usage of it in place as well. Is it being used somewhere?

Reply:
API 👍